src/CapabilityService.WebApi/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#Setup build image
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build-env
+
+#Set container filesystem to /build (and create folder if it doesnt exist)
+WORKDIR /build
+
+#Copy files to container file system.
+COPY . .
+
+#Set workdir to current project folder
+WORKDIR /build/src/CapabilityService.WebApi
+
+#Restore csproj packages.
+RUN dotnet restore
+
+#Compile source code using standard Release profile
+RUN dotnet publish -c Release -o /build/out
+
+#Setup final container images.
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2 AS runtime
+WORKDIR /app
+
+#Copy binaries from publish container to final container
+COPY --from=build-env /build/out .
+
+# SSL
+ENV CAPABILITY_SERVICE_KAFKA_SSL_CA_LOCATION=/app/cert.pem
+
+RUN curl -o /tmp/rds-combined-ca-bundle.pem https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem \
+    && mv /tmp/rds-combined-ca-bundle.pem /usr/local/share/ca-certificates/rds-combined-ca-bundle.crt \
+    && update-ca-certificates
+
+RUN curl -sS -o /app/cert.pem https://curl.haxx.se/ca/cacert.pem
+
+#Run dotnet executable
+ENTRYPOINT ["dotnet", "CapabilityService.WebApi.dll"]
```

Comment:
I think your feeling is a good one, it would make sense to make an ADR, but quiet frankly atm I fear our ADR wiki is so polluted it might drown out the good ideas, so maybe adding a section about it in our guidelines would make more sense? @ https://github.com/dfds/cag/blob/master/docs/guidelines/microservices/CONTAINERIZATION.md. 

Also I have noticed that building inside images seem to be the "visual studio standard" these days and I dont really bother to much about build times, as long as it doesnt exceed an hour, im more concerned about people building their images with a focus on size so its quick to push across the network. :)